### PR TITLE
Use logo.svg in the README

### PR DESCRIPTION
**Pull request by jacobherrington**
I noticed that we don't use the SVG in the readme.

Here is the difference:

screen shot 2018-10-03 at 5 37 39 am

### CODE REVIEW

**Comment by jacobherrington** [comment]
We can copy the SVG over to the root of this repo if that's better?

File: README.md [hunk 1]
```diff
@@ -1,5 +1,5 @@
 
-![](https://raw.githubusercontent.com/solidusio/solidus/master/solidus.png)
+<img src="./logo.svg" width=350>
```
Comment (tvdeyen) [inline README.md]: Why don't we use markdown anymore?

Comment (ericsaupe) [reply README.md]: Regular HTML works in markdown but I'd prefer the markdown version too.

Comment (ericsaupe) [reply README.md]: AFAIK there isn't a way to manipulate size using Markdown.

Comment (ericsaupe) [reply README.md]: Yeah, I usually prefer Markdown over Markdown + HTML but I'm not totally against having HTML in the README. Is it worth it/possible to resave the SVG to a smaller size or will it simply enlarge itself by default of how the Markdown is rendered? Just thinking of other options if we really were set on not doing the HTML

Comment (jacobherrington) [reply README.md]: It's pretty huge, but if that's okay markdown works

**Pull request merged**